### DISCUSSION TITLE
Welcome to RaspiWiFi Discussions!

### DISCUSSION THREAD

**Discussion by jasbur** [jasbur/RaspiWiFi#122] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bizzleboy** [comment]
when i install this on my raspberry pi 3 a+ running diet pi, it creates the network but i cant connect to any of the links and then also can no longer connect to wifi again on my pi, making me restart and delete EVERYTHing
